Remotion/ObjectBinding/Web/UI/Controls/BocTreeView.cs

```diff
@@ -974,12 +994,13 @@ public BusinessObjectPropertyTreeNodeInfo (IBusinessObjectReferenceProperty prop
       _property = property;
     }
 
-    public BusinessObjectPropertyTreeNodeInfo (string text, string toolTip, IconInfo icon, IBusinessObjectReferenceProperty property)
+    public BusinessObjectPropertyTreeNodeInfo (string text, string toolTip, IconInfo icon, Badge badge, IBusinessObjectReferenceProperty property)
```

Comment:
The design isn't all that great there but we can't go and rewrite everything so the solution is we drop the assignment from the ctor here and instead set the Badge-property for BusinessObjectProeprties right after we create the BusinessObjectPropertyTreeNodeInfo  in BocTreeView via `nodeInfo = ctor(); nodeInfo.Badge=GetBadge(property);`